packages/geo/src/lib/catalog/catalog-browser/catalog-browser-layer.component.html

```diff
@@ -1,17 +1,25 @@
 <mat-list-item>
   <mat-icon mat-list-avatar svgIcon="blank"></mat-icon>
-  <h4 mat-line>{{title}}</h4>
+  <h4 mat-line matTooltipShowDelay="500" [matTooltip]="title">{{title}}</h4>
 
   <igo-metadata-button [layer]="layer"></igo-metadata-button>
 
   <button
     mat-icon-button
     tooltip-position="below"
     matTooltipShowDelay="500"
-    [matTooltip]="(added ? 'igo.geo.catalog.layer.removeFromMap' : 'igo.geo.catalog.layer.addToMap') | translate"
+    [matTooltip]="tooltip | translate"
     [color]="added ? 'warn' : ''"
     (click)="onToggleClick()">
-    <mat-icon [svgIcon]="added ? 'delete' : 'plus'"></mat-icon>
+    <mat-icon
+       matBadge
+       igoMatBadgeIcon="eye-off"
```

Comment:
Why add the igoMatBadgeIcon and the [svgIcon] attribute? I guess only one of them is needed?

Reply:
No The svg stills there as the "host" mat-icon.  The directive add a badge, containing itself a mat-icon too.